xbmc/guilib/GUIMoverControl.h

```diff
@@ -41,10 +41,6 @@
 #define DIRECTION_LEFT 3
 #define DIRECTION_RIGHT 4
 
-// normal alignment is TOP LEFT 0 = topleft, 1 = topright
-#define ALIGN_RIGHT   1
```

Comment:
They clash with something in cppformat and they're not used anywhere that I could find so I just dropped them